packages/config/plugins/write-manifest.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var mkdirp = require('mkdirp');
+
+var hopsConfig = require('..');
+
+module.exports = function Plugin (regExp) {
+  mkdirp.sync(hopsConfig.cacheDir);
+  this.apply = function (compiler) {
+    compiler.plugin('emit', function (compilation, callback) {
+      var paths = Object.keys(compilation.assets);
+      var assets = paths.reduce(function (assets, path) {
+        if (path.indexOf('hot-update') >= 0 || path.indexOf('chunk-') >= 0) {
+          return assets;
+        }
+        if (path.indexOf('.css') === path.length - 4) {
```

Comment:
`path` is a shadowed variable here which makes it harder to understand. 

Did you think about using `path.extname(assetPath) === '.css')`? That could make the intend a bit more obvious.